### DISCUSSION TITLE
How to add the X Axis label to a Tooltip

### DISCUSSION THREAD

**Discussion by grewegreg** [Live-Charts/LiveCharts2#588] [Q&A]
When a Tooltip is displayed, can the X Axis label be displayed as the Title in the Tooltip?  Such as:

### Title:

- Series.Name PrimaryValue
- Series.Name PrimaryValue
- Series.Name PrimaryValue
- Series.Name PrimaryValue
- Series.Name PrimaryValue

### Example:
![LiveCharts2TooltipQuestion](https://user-images.githubusercontent.com/91921107/185632597-6b4a0e48-1596-46f7-9f88-d66501b95a50.png)

The code that populates the chart is listed below.   In the graphic above, _PropertyStatus_ is the property from the Model that contains the X Axis values.
```
 IList<ColumnSeries<decimal>> columnSeries = new List<ColumnSeries<decimal>>();
        foreach (short taxYear in value
            .OrderBy(o => o.TaxYear)
            .ThenBy(t => t.PropertyStatus)
            .Select(s => s.TaxYear).Distinct())
        {
            columnSeries.Add(new()
            {
                Name = taxYear.ToString(),
                Values = value.Where(w => w.TaxYear.Equals(taxYear)).Select(s => s.AvgDonation),
                TooltipLabelFormatter = (chartPoint) => $"{chartPoint.Context.Series.Name}: {chartPoint.PrimaryValue:C0}",
            });
        }
        Series = columnSeries.ToArray();

        XAxes = new Axis[]
        {
            new Axis
            {
                Labels = value
                .OrderBy(o => o.PropertyStatus)
                .Select(s => s.PropertyStatus.Substring(0, 6))
                .Distinct().ToList(),
                //LabelsRotation = 15
            }
        };

        YAxes = new Axis[]
        {
            new Axis
            {
                Labeler = Labelers.Currency
            }
        };
```

**Comment by Panache1** [comment]
This will get you close.  SecondaryValue gets you the index number for the array of values.

I haven't figured out a way to add a new line in between, though.

`TooltipLabelFormatter = (chartPoint) => $"{values[(int)chartPoint.SecondaryValue].PropertyStatus}" + $": chartPoint.Context.Series.Name}: {chartPoint.PrimaryValue:C0}"
`

**Reply by Ethilyk** [reply]
If you cast chart as CartesianChart<SkiaSharpDrawingContext> you can access the Xaxes.Labels using the SecondaryValue from the point as an index
